x/evm/types/msg_test.go

```diff
@@ -91,23 +89,6 @@ func (suite *MsgsTestSuite) TestMsgEthereumTx_ValidateBasic() {
 	}
 }
 
-func (suite *MsgsTestSuite) TestMsgEthereumTx_EncodeRLP() {
```

Comment:
what do about RLP?

Reply:
the correct way to encode is to use `msg.AsTransaction().MarshalBinary()`